openff/toolkit/topology/molecule.py

```diff
@@ -3406,7 +3419,7 @@ def nth_degree_neighbors(self, n_degrees):
         Returns
         -------
         neighbors: iterator of tuple of Atom
-            Tuples (len 2) of atom that are separated by ``n`` bonds.
+            tuples (len 2) of atom that are separated by ``n`` bonds.
```

Comment:
```suggestion
            Tuples (len 2) of atom that are separated by ``n`` bonds.
```

Overzealous find and replace?

Reply:
Yeah, I thought I caught all of these